docs/pages/eas/hosting/custom-domain.mdx

```diff
@@ -44,7 +44,47 @@ You will need to own a domain name you want to use.
 5. Press the refresh button until all checks pass. Depending on your DNS provider, this step usually only takes a couple of minutes.
 
 > If you require for the domain name switchover to be **zero downtime**, it's important to fill out these records one by one in the order they are presented in the table.
-> That is, add the **TXT** record for verification, press the refresh button until the UI says verification is successful, then proceed to the next one.
+> That is, add the **Verification TXT** record first, and press "Refresh" until the UI confirms the verification record. Then add the **SSL CNAME** record next until
+> it is confirmed, and set up the third record last.
 > If downtime isn't important or relevant, you may add all three DNS records at once.
 
 After assigning a custom domain to your app, the custom domain will route to your **production** deployment.
+
+### Custom domain DNS records
+
+Two of the three records the dashboard presents you are to validate ownership of your domain.
+The **Verification TXT** record proves ownership of your domain, since it adds a custom token that can be read back to verify you're setting the domain up on a domain you control.
+The **SSL CNAME** record proves ownership of your domain to a certificate authority, also known as Domain Control Validation (DCV). This is a CNAME record because both renewal and validation is delegated to an automated process, which prevents certificates from expiring.
+
+Both records are created on a subdomain of the custom domain you're setting up.
+
+- If you're setting up `example.com`, the records must be created on `_cf-custom-hostname.example.com` and `_acme-challenge.example.com` respectively
+- If you're setting up `anything.example.com`, the records must be created on `_cf-custom-hostname.anything.example.com` and `_acme-challenge.example.com` respectively
```

Comment:
Just checking that the `_acme-challenge` doesn't belong on the subdomain. Naively I'd assume you'd want this so that different subdomains can have their own challenges:
```suggestion
- If you're setting up `anything.example.com`, the records must be created on `_cf-custom-hostname.anything.example.com` and `_acme-challenge.anything.example.com` respectively
```